automation/vm/deploy-vm.ps1

```diff
@@ -0,0 +1,70 @@
+[CmdletBinding()]
+param (
+    [Parameter(Mandatory=$True)]
+    [string]
+    $SubscriptionId,
+
+    [Parameter(Mandatory=$True)]
+    [string]
+    $TenantId,
+
+    [Parameter(Mandatory=$False)]
+    [string]
+    $ServicePrincipalId,
```

Comment:
this parameter can be removed since it is not being used in the script

Reply:
I will keep it for now in case we all need to use it, it's in "false" state so not required